Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -255,6 +252,32 @@ class PhotonActionSheet: UIViewController {
         view.insertSubview(imageView, belowSubview: tableView)
     }
 
+    func applyTheme() {
+        let theme = themeManager.currentTheme
+
+        // In a popover the popover provides the blur background
+        if viewModel.presentationStyle == .popover {
+            view.backgroundColor = theme.colors.layer1.withAlphaComponent(0.7)
+        } else if UIAccessibility.isReduceTransparencyEnabled {
+            // Remove the visual effect and the background alpha
+            (tableView.backgroundView as? UIVisualEffectView)?.effect = nil
+            tableView.backgroundView?.backgroundColor = theme.colors.layer1.withAlphaComponent(0.9)
+        } else {
+            let blurEffect = theme.type == .dark ? UIBlurEffect(style: .dark) : UIBlurEffect(style: .light)
```

Comment:
Noted this is a different blur style from the homepage, that's expected? 
Also, if that's expected then let's replace .`theme.type == .dark` with some equivalent of `ThemeType.getBarStyle()`, so we have some equivalent of the following:
```
let blurEffect = UIBlurEffect(style: theme.type.blurStyle)
```
This will easily support if we end up having new theme types in the future.